javascript/javascript1/week1/goodboy-oldboy.js

```diff
@@ -0,0 +1,21 @@
+const dogYearOfBirth = 2020;
+const dogYearFuture = 2030;
+let shouldShowResultInDogYears = true;
```

Comment:
could be const.

Reply:
will keep in mind in next time,thank you.